docs/resources/aws_ec2_vpn_connection_routes.md

```diff
@@ -0,0 +1,66 @@
+---
+title: About the aws_ec2_vpn_connection_routes Resource
+platform: aws
+---
+
+# aws_ec2_vpn_connection_routes
+
+Use the `aws_ec2_vpn_connection_routes` InSpec audit resource to test properties of a plural AWS EC2 VPN Connection Route.
+
+The AWS::EC2::VPNConnectionRoute resource specifies a static route for a VPN connection between an existing virtual private gateway and a VPN customer gateway. The static route allows traffic to be routed from the virtual private gateway to the VPN customer gateway.
+
+## Syntax
+
+Ensure that VPN Connection ID exists.
+
+    describe aws_ec2_vpn_connection_routes do
```

Comment:
Does this require a VPN connection ID?

Reply:
No

Reply:
Is it an optional parameter?

Reply:
NO

Reply:
Then why is it on line 10 of libraries/aws_ec2_vpn_connection_routes.rb:
`describe aws_ec2_vpn_connection_routes(vpn_connection_id: 'VPNConnectionID') do`
Also lines 25 and 26?

Reply:
vpn_connection_id is required in the plural resource. I have added the same in the documentation.